automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoTestUtils.scala

```diff
@@ -108,8 +110,11 @@ trait LeonardoTestUtils extends WebBrowserSpec with Matchers with Eventually wit
     expectedStatuses should contain (cluster.status)
     cluster.googleProject shouldBe expectedProject
     cluster.clusterName shouldBe expectedName
-    labelCheck(cluster.labels, requestedLabels, expectedName, expectedProject, cluster.creator, notebookExtension)
+    cluster.stagingBucket shouldBe 'defined
 
+    implicit val patienceConfig: PatienceConfig = storagePatience
+    googleStorageDAO.bucketExists(google.GcsBucketName(cluster.stagingBucket.get.value)).futureValue shouldBe true
```

Comment:
This doesn't need to be `google.GcsBucketName(...)`, it can just be `GcsBucketName`